2021-09-17 17:14:23,413 INFO    MainThread:5026 [wandb_setup.py:_flush():69] setting env: {}
2021-09-17 17:14:23,413 INFO    MainThread:5026 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-17 17:14:23,413 INFO    MainThread:5026 [wandb_setup.py:_flush():69] setting login settings: {'anonymous': 'allow'}
2021-09-17 17:14:23,413 INFO    MainThread:5026 [wandb_init.py:_log_setup():348] Logging user logs to /home/denis/AI/python/clasificador de imagen/kaggle brain tumor/src/wandb/run-20210917_171423-3qni86yp/logs/debug.log
2021-09-17 17:14:23,413 INFO    MainThread:5026 [wandb_init.py:_log_setup():349] Logging internal logs to /home/denis/AI/python/clasificador de imagen/kaggle brain tumor/src/wandb/run-20210917_171423-3qni86yp/logs/debug-internal.log
2021-09-17 17:14:23,414 INFO    MainThread:5026 [wandb_init.py:init():381] calling init triggers
2021-09-17 17:14:23,414 INFO    MainThread:5026 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {'MAX_FRAMES': 12, 'EPOCHS': 5, 'LR': 0.0002, 'IMG_SIZE': (224, 224), 'FEATURE_EXTRACTOR': 'resnext50_32x4d', 'DR_RATE': 0.35, 'NUM_CLASSES': 1, 'RNN_HIDDEN_SIZE': 100, 'RNN_LAYERS': 1, 'TRAIN_BS': 4, 'VALID_BS': 4, 'NUM_WORKERS': 1, 'infra': 'Kaggle', 'competition': 'rsna_miccai', '_wandb_kernel': 'tanaym'}
2021-09-17 17:14:23,414 INFO    MainThread:5026 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 3t1yt44s
2021-09-17 17:14:23,415 INFO    MainThread:5026 [wandb_run.py:finish():1275] finishing run denisquant/kaggle_brain_tumor/3t1yt44s
2021-09-17 17:14:23,415 INFO    MainThread:5026 [jupyter.py:save_history():353] not saving jupyter history
2021-09-17 17:14:23,415 INFO    MainThread:5026 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-17 17:14:23,416 INFO    MainThread:5026 [wandb_init.py:_jupyter_teardown():281] cleaning up jupyter logic
2021-09-17 17:14:23,416 INFO    MainThread:5026 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-09-17 17:14:23,416 INFO    MainThread:5026 [wandb_run.py:_restore():1547] restore
2021-09-17 17:14:23,546 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 715
}

2021-09-17 17:14:23,849 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 715
}

2021-09-17 17:14:24,860 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 13256
}

2021-09-17 17:14:24,963 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 13256
}

2021-09-17 17:14:25,071 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 13256
}

2021-09-17 17:14:25,172 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}

2021-09-17 17:14:25,274 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}

2021-09-17 17:14:25,376 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}

2021-09-17 17:14:25,479 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}

2021-09-17 17:14:25,580 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}

2021-09-17 17:14:25,682 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}

2021-09-17 17:14:26,086 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}

2021-09-17 17:14:26,554 INFO    MainThread:5026 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13256
  total_bytes: 13256
}
local_info {
}

2021-09-17 17:14:27,655 INFO    MainThread:5026 [wandb_run.py:_show_files():1947] logging synced files
2021-09-17 17:14:27,660 INFO    MainThread:5026 [wandb_init.py:init():430] starting backend
2021-09-17 17:14:27,660 INFO    MainThread:5026 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-17 17:14:27,665 INFO    MainThread:5026 [backend.py:ensure_launched():135] starting backend process...
2021-09-17 17:14:27,694 INFO    MainThread:5026 [backend.py:ensure_launched():139] started backend process with pid: 7202
2021-09-17 17:14:27,696 INFO    MainThread:5026 [wandb_init.py:init():435] backend started and connected
2021-09-17 17:14:27,716 INFO    MainThread:5026 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-09-17 17:14:27,716 INFO    MainThread:5026 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-17 17:14:27,716 INFO    MainThread:5026 [wandb_init.py:init():494] updated telemetry
2021-09-17 17:14:27,717 INFO    MainThread:5026 [wandb_init.py:init():517] communicating current version
2021-09-17 17:14:28,291 INFO    MainThread:5026 [wandb_init.py:init():522] got version response 
2021-09-17 17:14:28,291 INFO    MainThread:5026 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-17 17:14:28,526 INFO    MainThread:5026 [wandb_init.py:init():557] starting run threads in backend
2021-09-17 17:14:30,929 INFO    MainThread:5026 [wandb_run.py:_console_start():1605] atexit reg
2021-09-17 17:14:30,930 INFO    MainThread:5026 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-09-17 17:14:30,930 INFO    MainThread:5026 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-09-17 17:14:30,933 INFO    MainThread:5026 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-17 17:14:30,933 INFO    MainThread:5026 [wandb_init.py:init():582] run started, returning control to user process
2021-09-17 17:19:25,926 INFO    MainThread:5026 [wandb_setup.py:_flush():69] setting env: {}
2021-09-17 17:19:25,926 INFO    MainThread:5026 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-17 17:19:25,926 INFO    MainThread:5026 [wandb_setup.py:_flush():69] setting login settings: {'anonymous': 'allow'}
2021-09-17 17:19:25,926 INFO    MainThread:5026 [wandb_init.py:_log_setup():348] Logging user logs to /home/denis/AI/python/clasificador de imagen/kaggle brain tumor/src/wandb/run-20210917_171925-hj814c2u/logs/debug.log
2021-09-17 17:19:25,927 INFO    MainThread:5026 [wandb_init.py:_log_setup():349] Logging internal logs to /home/denis/AI/python/clasificador de imagen/kaggle brain tumor/src/wandb/run-20210917_171925-hj814c2u/logs/debug-internal.log
2021-09-17 17:19:25,927 INFO    MainThread:5026 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1c083c45e0>
2021-09-17 17:19:25,928 INFO    MainThread:5026 [wandb_init.py:init():381] calling init triggers
2021-09-17 17:19:25,928 INFO    MainThread:5026 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {'MAX_FRAMES': 12, 'EPOCHS': 5, 'LR': 0.0002, 'IMG_SIZE': (224, 224), 'FEATURE_EXTRACTOR': 'resnext50_32x4d', 'DR_RATE': 0.35, 'NUM_CLASSES': 1, 'RNN_HIDDEN_SIZE': 100, 'RNN_LAYERS': 1, 'TRAIN_BS': 4, 'VALID_BS': 4, 'NUM_WORKERS': 4, 'infra': 'Kaggle', 'competition': 'rsna_miccai', '_wandb_kernel': 'tanaym'}
2021-09-17 17:19:25,928 INFO    MainThread:5026 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 3qni86yp
2021-09-17 17:19:25,930 INFO    MainThread:5026 [wandb_run.py:finish():1275] finishing run denisquant/kaggle_brain_tumor/3qni86yp
